desktop/api/src/main/java/org/datacleaner/panels/AbstractComponentBuilderPanel.java

```diff
@@ -163,6 +165,18 @@ private final void init() {
         onOutputDataStreamsChanged();
     }
 
+    private void addInformationPanelAboutDisable() {
+        if (_descriptor instanceof Allowable && !((Allowable) _descriptor).isAllowed()) {
+            final ImageIcon icon = imageManager.getImageIcon(IconUtils.STATUS_INFO, IconUtils.ICON_SIZE_TASK_PANE);
+            DCPanel panel = new DCPanel();
+            DCHtmlBox dcHtmlBox = new DCHtmlBox(
+                    "<p>This service is not currently available for your DataCloud account.</p>"
+                            + "<p> Please contact 'support@datacleaner.org' to obtain access.</p>");
+            panel.add(dcHtmlBox);
+            addTaskPane(icon, "Component is disabled", panel, true);
```

Comment:
We have a standard for these panels. Please don't make everything to HTML boxes.